automatic/opera/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,27 @@
+$ErrorActionPreference = 'Stop'
+
+$packageName         = 'opera'
```

Comment:
Recommendation: You could actually use `env:ChocolateyPackageName` as the value.